kevm

```diff
@@ -53,8 +54,15 @@ run_krun() {
     local backend=$1  ; shift
     local run_file=$1 ; shift
     run_env "$run_file"
-    export K_OPTS=-Xss500m
-    krun --directory "$build_dir/$backend/" -cSCHEDULE="$cSCHEDULE" -pSCHEDULE='printf %s' -cMODE="$cMODE" -pMODE='printf %s' "$run_file" "$@"
+    if [[ "$backend" == "haskell" ]]; then
+        kast="$(mktemp "$build_dir/haskell/kast.XXXXXX")"
+        trap "rm -rf $kast" INT TERM EXIT
+        kast --directory "$build_dir/haskell" --sort K --kore $run_file > $kast
+        stack exec -- kore-exec "$build_dir/haskell/driver.kore" --module ETHEREUM-SIMULATION --is-program --pattern $kast
```

Comment:
I can't see where `haskell/driver.kore` is created, but I recall that getting the correct result here depends on the options to `kompile`.

Reply:
`haskell/driver.kore` should be created by the `Makefile`. The `./kevm` script assumes in all cases that you've built using the `Makefile` prior to use.